### ISSUE TITLE
hmm_frame assertion fails

### ISSUE THREAD

**Issue by sbraden** [cmusphinx/pocketsphinx#9]
I'm on OS X Yosemite, 10.10.2. 

I am running the following code and getting this error:

```
Assertion failed: (hmm_frame(&hmm->hmm) == frame_idx), function eval_nonroot_chan, file ngram_search_fwdtree.c, line 640.
```

I installed two different versions of sphinxbase and pocketsphinx, once directly from the git repos, and once from the 5prealpha files on sourceforge. Both give the same error. 

Pocketsphinx_continuous runs just fine on both installs. This is my first time using sphinx, so any help you can offer would be appreciated. I looked at the source code line that is causing the error, but I am drawing a blank on how to fix it.

This user is the only other person I know who has had this error: https://github.com/watsonbox/sphinxtrain-ruby/issues/3

I haven't tried pocketsphinx version 0.8 yet. 

Sample code:

    #!/usr/bin/ python

    import sys
    from os import path
    import pocketsphinx

    base_dir = '/usr/local/share/pocketsphinx/model/en-us'
    hidden_markov_model = path.join(base_dir, 'en-us')  #acoustic model
    language_model = path.join(base_dir, 'cmusphinx-5.0-en-us.lm')
    dictionary = path.join(base_dir, 'cmudict-en-us.dict')

    config = pocketsphinx.Decoder.default_config()
    config.set_string('-hmm', hidden_markov_model)
    config.set_string('-dict', dictionary)
    config.set_string('-lm', language_model)

    decoder = pocketsphinx.Decoder(config)
    decoder.start_utt()
    wavfile = sys.argv[1]
    wav_file = file(wavfile, 'rb')
    decoder.decode_raw(wav_file)
    result = decoder.hyp().hypstr

    decoder.end_utt()

    print result

**Comment by nshmyrev** [comment]
You should not start utt before decode_raw, it starts utt already, duplicate of the issue https://sourceforge.net/p/cmusphinx/bugs/405/

**Comment by nshmyrev** [comment]
So you basically need to remove decoder.start_utt() and decoder.start_utt(). Also you can update from github, it now will issue a error instead of crash
